### DISCUSSION TITLE
openresty 重定向问题

### DISCUSSION THREAD

**Discussion by byhgj** [1Panel-dev/1Panel#4147] [Q&A]
在使用openresty(好像1panel只能用它了)，在网站有302时会连续重定向，且错误重定向，比如安装zblog时，/index.php -> ./zb_install/index.php -> ./zb_install/zb_install/index.php   然后报错404，使用caddy正常。怀疑是lua waf引起，没法测试（也可能不会）。有没有谁遇到此现象，怎么解决？？

**Comment by zhengkunwang223** [comment]
> 在使用openresty(好像1panel只能用它了)，在网站有302时会连续重定向，且错误重定向，比如安装zblog时，/index.php -> ./zb_install/index.php -> ./zb_install/zb_install/index.php 然后报错404，使用caddy正常。怀疑是lua waf引起，没法测试（也可能不会）。有没有谁遇到此现象，怎么解决？？

所有的网站都有这个问题 还是安装特定应用的有这个问题？
